eos.doxygen.in

```diff
@@ -4,8 +4,8 @@
 # Project related configuration options
 #---------------------------------------------------------------------------
 DOXYFILE_ENCODING      = UTF-8
-PROJECT_NAME           = "EOS.IO"
-PROJECT_NUMBER         = "EOSIO ${DOXY_EOS_VERSION}"
+PROJECT_NAME           = "ANTELOPE.IO"
+PROJECT_NUMBER         = "LEAP ${DOXY_EOS_VERSION}"
```

Comment:
I would think that `PROJECT_NAME` would just be `"Leap"` and `PROJECT_NUMBER` would just be `"{DOXY_EOS_VERSION}"`.

Reply:
Ok will change.

Reply:
fixed in commit https://github.com/AntelopeIO/leap/pull/358/commits/ae0000512727a7ae6af24b2451d9ebe5d1dbd9e0.